modules/ROOT/pages/notifications/index.adoc

```diff
@@ -1,35 +1,203 @@
+- `INFORMATION`: The query is correct, but this information can still be useful.
+|Category
+a|The category of the notification.
+|Position
+a|The position, given by row and column, where the notification is relevant in the query text.
+|===
+
+
+[role=label--version-5.23]
+[[gqlstatus-notification-object]]
+== GQL-status notification object
+
+The Neo4j GQL-status notifications are an implementation-defined subset of the GQL-status objects that cover informational notes and warnings, but not errors, `SUCCESS`, `NO DATA`, or `OMITTED RESULT`.
+For the latter, see <<general-codes-for-success, General codes for success>>.
+
+Each GQL-status object contains diagnostic information about the execution outcome.
+This information includes the GQLSTATUS code, the execution outcome (successful completion, warnings, no data, information), the status description, and the diagnostic record. +
+The main GQL-status object, describing the condition with the greatest precedence, is called the primary GQL-status object.
+The primary GQL-status object is always present in the execution outcome and follows the GQL precedence rules:
+
+. `NO DATA` has precedence over `WARNING`.
+. `WARNING` has precedence over the `SUCCESSFUL COMPLETION` subclass.
+. `SUCCESSFUL COMPLETION` subclass has precedence over `INFORMATIONAL`.
+. `INFORMATIONAL` is the condition with the least precedence.
+
+The primary GQL-status object also includes the Neo4j-specific information, such as the severity level and the classification of the notification, which can be used for filtering.
+For more information about notification grouping and filtering, see <<notification-grouping-and-filtering>>.
+
+The additional GQL-status objects are part of the list of additional GQL-status objects of the execution outcome.
+The list does not follow any specific order and might be empty as well.
+
+The GQL-status object consists of the following fields:
+
+.GQLSTATUS notification object
+[cols="<1s,<4"]
+|===
+|GQLSTATUS code
+a| A 5-character string that is the concatenation of a 2-character class code followed by a 3-character subclass code.
+|GQLSTATUS description
+a| A human-readable description of the GQLSTATUS, which consists of a condition, a subcondition, and a description.
+The condition and subcondition are the class and subclass codes, respectively.
+| Diagnostic record
+a| Extra information about the status, given as key-value pairs, both on the server and driver side.
+The full diagnostic record can be returned by the `getDiagnosticRecord()` method on the server side and the `diagnosticRecord()` methods on the driver side.
+Additional helper methods are exposed for some useful fields.
+[cols="<1s,<4"]
+!===
+! Field ! Description
+! OPERATION ! The operation that the notification is related to.
+! OPERATION_CODE ! The operation code that the notification is related to.
+! CURRENT_SCHEMA ! The current schema that the notification is related to.
```

Comment:
We might want to add a note saying that we only have these because GQL requires it and that they will always just give a default value and therefore are pointless. You might want to phrase that in a nicer way though xD

Reply:
Do we want them documented, then?

Reply:
I'm a bit wary to say that we don't use them.

Reply:
I think we have to in order to be compliant. I think what you wrote now looks good